From f4b31ce4b67e8e4987008fa547a8402d609e55ab Mon Sep 17 00:00:00 2001
From: Fabio Berton <fabio.berton@ossystems.com.br>
Date: Fri, 7 Jun 2019 17:56:52 -0300
Subject: [PATCH 1/2] setup.py: remove the setup_requires for setuptools-scm
Organization: O.S. Systems Software LTDA.

Use the same patch that pytest recipe from meta-python layer:

https://git.openembedded.org/meta-openembedded/plain/meta-python/recipes-devtools/python/python-pytest/0001-setup.py-remove-the-setup_requires-for-setuptools-scm.patch?h=warrior

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Fabio Berton <fabio.berton@ossystems.com.br>

%% original patch: 0001-Remove-setuptools_scm-from-setup_requires.patch
---
 setup.cfg | 1 -
 1 file changed, 1 deletion(-)

diff --git a/setup.cfg b/setup.cfg
index 5fb5030..1082fca 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -22,7 +22,6 @@ packages = find:
 include_package_data = true
 python_requires = >=2.7
 install_requires = 
-setup_requires = setuptools_scm >= 1.15.0
 
 [options.extras_require]
 testing = 
-- 
2.20.1

